target/linux/mvebu/image/Makefile

```diff
@@ -226,4 +241,21 @@ define Device/globalscale-mirabox
 endef
 TARGET_DEVICES += globalscale-mirabox
 
+define Device/turris-omnia
+  KERNEL_INSTALL := 1
+  KERNEL := kernel-bin
+  KERNEL_INITRAMFS := kernel-bin
+  DEVICE_TITLE := Turris Omnia
+  DEVICE_PACKAGES :=  \
+    mkf2fs e2fsprogs kmod-fs-vfat kmod-nls-cp437 kmod-nls-iso8859-1 \
+    wpad-mini kmod-ath9k kmod-ath10k ath10k-firmware-qca988x
+  IMAGES := $$(IMAGE_PREFIX)-sysupgrade.img.gz omnia-medkit-$$(IMAGE_PREFIX)-initramfs.tar.gz
+  IMAGE/$$(IMAGE_PREFIX)-sysupgrade.img.gz := boot-scr-dummy | boot-img | sdcard-img | gzip | append-metadata
+  IMAGE/omnia-medkit-$$(IMAGE_PREFIX)-initramfs.tar.gz := omnia-medkit-initramfs | gzip
+  IMAGE_NAME = $$(2)
+  DEVICE_DTS := armada-385-turris-omnia
+  UBOOT :=
```

Comment:
Drop the empty variable.

Reply:
It won't build without this being unset, otherwise for some reason it remains clearfog-u-boot-spl.kwb and sdcard-img goes looking for a nonexistent file.

Reply:
OK, after I get some sleep, I'll check why it's failing.

Reply:
On my side the build didn't fail (removed tmp and .config for each build), but what I observed is that when UBOOT is not specified it inherits the variable from previous device embedding the clearfog bootloader to the image. When it was set like in Your PR it made proper image without bootloader.
Seem there's non fatal bug in the bootloader handling code I introduced, I'll fix it in the future. Your approach is correct.